docs_user/modules/proc_adopting-the-orchestration-service.adoc

```diff
@@ -2,64 +2,57 @@
-will then have CRs for `Heat`, `HeatAPI`, `HeatEngine` and `HeatCFNAPI`.
-Additionally, a user should have endpoints created within {identity_service_first_ref} to facilitate
-the above mentioned servies.
-
-This guide also assumes that:
-
-. A {OpenStackPreviousInstaller} environment (the source Cloud) is running on one side;
-. A {rhocp_long} environment is running on the other side.
+After you complete the adoption process, you have CRs for `Heat`, `HeatAPI`, `HeatEngine`, and `HeatCFNAPI`, and endpoints within the {identity_service_first_ref} to facilitate these services.
 
 .Prerequisites
 
-* Previous Adoption steps completed. Notably, MariaDB and {identity_service}
-should be already adopted.
-* In addition, if your existing {orchestration} stacks contain resources from other services
-such as {networking_first_ref}, {compute_service_first_ref}, {object_storage_first_ref}, etc. Those services should be adopted first before
-trying to adopt {orchestration}.
+* The source {OpenStackPreviousInstaller} environment is running.
+* The target {rhocp_long} environment is running.
+* You adopted MariaDB and the {identity_service}.
+* If your existing {orchestration} stacks contain resources from other services such as {networking_first_ref}, {compute_service_first_ref}, {object_storage_first_ref}, and so on, adopt those sevices before adopting the {orchestration}.
 
 .Procedure
 ifeval::["{build}" != "downstream"]
 As already done for https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/keystone_adoption.md[Keystone], the Heat Adoption follows a similar pattern.
 endif::[]
 
-. Patch the `osp-secret` to update the `HeatAuthEncryptionKey` and `HeatPassword`. This needs to match what you have configured in the existing {OpenStackPreviousInstaller} {orchestration} configuration.
-You can retrieve and verify the existing `auth_encryption_key` and `service` passwords via:
+. Retrieve the existing `auth_encryption_key` and `service` passwords. You use these passwords to patch the `osp-secret`:
 +
 ----
 [stack@rhosp17 ~]$ grep -E 'HeatPassword|HeatAuth' ~/overcloud-deploy/overcloud/overcloud-passwords.yaml
   HeatAuthEncryptionKey: Q60Hj8PqbrDNu2dDCbyIQE2dibpQUPg2
   HeatPassword: dU2N0Vr2bdelYH7eQonAwPfI3
 ----
 
-. And verifying on one of the Controllers that this is indeed the value in use:
+. Log in to a Controller node and verify that the `auth_encryption_key` value is the same as the `HeatAuthEncryptionKey` value:
 +
 ----
 [stack@rhosp17 ~]$ ansible -i overcloud-deploy/overcloud/config-download/overcloud/tripleo-ansible-inventory.yaml overcloud-controller-0 -m shell -a "grep auth_encryption_key /var/lib/config-data/puppet-generated/heat/etc/heat/heat.conf | grep -Ev '^#|^$'" -b
 overcloud-controller-0 | CHANGED | rc=0 >>
 auth_encryption_key=Q60Hj8PqbrDNu2dDCbyIQE2dibpQUPg2
 ----
 
-. This password needs to be base64 encoded and added to the `osp-secret`
+. Base64 encode the password:
```

Comment:
```suggestion
. Encode the password to Base64 format:
```

This is something very minor, maybe I'm being too pedantic here.

Reply:
I split this step into 2 steps. Step 3 is converting to base64 format, and step 4 is patching the osp-secret.